CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# How to contribute?
+
+## Testing 
+
+Just use OpenTracks and tell us what can be improved by opening an issue [here](https://github.com/OpenTracksApp/OpenTracks/issues/new/choose)
+
+If we broke things (happens now and then), please tell us also.
+
+## Coding new features
+
+You have cool idea for a feature that is missing in OpenTracks?
+And you would like to help by implementing it?
+If not, you can just take a look at the [issue tracker](https://github.com/OpenTracksApp/OpenTracks/issues).
+
+Helpful hints:
+* creating an issue beforehand might be helpful
+  Sometimes it is helpful to discuss your idea and also your implementation approach first.
```

Comment:
Three times "helpful" in a row. 
Maybe "Tips" or "Advice" as the headline and "... might be good" in the second line.